addons/web_editor/static/src/js/backend.js

```diff
@@ -236,10 +236,13 @@ var FieldTextHtml = AbstractField.extend({
         // this.$translate = $();
         return def;
     },
+    get_datarecord: function () {
+        return this.recordData;
+    },
     get_url: function (_attr) {
         var src = this.nodeOptions.editor_url || "/mass_mailing/field/email_template";
         var k;
-        var datarecord = this.recordData;
+        var datarecord = this.get_datarecord();
```

Comment:
Could you camelcase the function? There are the new JS coding guidelines: https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines